packages/zarith/zarith.1.3/files/xen_linkopts.patch

```diff
@@ -2,10 +2,7 @@ diff --git a/META b/META
 index c082c65..14394d1 100644
 --- a/META
 +++ b/META
-@@ -1,5 +1,6 @@
- description = "Arbitrary precision integers"
--requires = ""
-+requires = "gmp"
+@@ -3,3 +3,4 @@
```

Comment:
I think something along the lines of https://github.com/mirage/mirage-dev/pull/56/files#diff-0eca2e8236c2b30dae11dde168158c5aR10 would work. ie. define sub-package `zarith.xen` which depends on `gmp-xen`. The only think is that then `zarith.xen` need to be pulled at the right place when needed...